README.md

```diff
@@ -144,6 +148,51 @@ go run ./cmd/lvh run --image _data/images/base.qcow2
 go run ./cmd/lvh run --image _data/images/base.qcow2 --qemu-disable-kvm
 ```
 
+### Cilium development workflow
+
+Set up an LVH VM based on the kind image:
+
+```bash
+./scripts/pull_image.sh quay.io/lvh-images/kind:bpf-next-main
+lvh run --image $(ls -Art images/*.qcow2 | tail -n 1) --host-mount /path/to/cilium
```

Comment:
Now you can just do `lvh run --image quay.io/lvh-images/kind:bpf-next-main --host-mount /path/to/cilium`. No need to separately pull.